guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,355 @@
+---
+layout: default
+title: Theme Quality Assurance
+subhead: Prior to launching any ecommerce site, review this list and verify the site is up to Engine standards.
+has_children: false
+parent: Guides
+nav_order: 5
+permalink: /guides/theme-qa/
+---
+
+{% include page_header.html %}
+
+By following these guidelines, theme builders can ensure their customers receive the world’s best eCommerce experience on the Engine platform.
+
+[This document is open-source]({{ site.repo_url }}) and is built to be expanded. By continually revising this document, we will ensure that problems are never encountered more than once. If you encounter problems, please [open an issue]({{ site.repo_issues_url }}) or [make a contribution]({{ site.repo_url }}).
+
+Much of this test and QA process can be automated. However, there is great value in human experience, so we recommend a **real** human being verify all final quality tests before launch.
+
+---
+
+##### Table of Contents
+{:.no_toc}
+
+* An unordered list to be replaced with the ToC, excluding the "##### Table of Contents" from above
+{:.no_toc}
+{:toc levels=1}
+
+---
+
+## Testing Environments
+
+To maximize ecommerce conversion rates, verify these tests in the following browser environments:
```

Comment:
This should also be written as "eCommerce" rather than "ecommerce" 😄

Reply:
Great catch!